### PR TITLE
After project refactoring merge fixes

### PR DESCRIPTION
**Pull request by wowkalucky**
Configuration fixes missed during project structure refactoring.

### CODE REVIEW

File: bridge_adaptivity/bridge_lti/urls.py [hunk 1]
```diff
@@ -4,6 +4,6 @@
 from .consumer import source_preview
 
 urlpatterns = [
-    url(r'^launch/(?P<collection_id>\d*)$', lti_launch, name='launch'),
```
Comment (wowkalucky) [inline bridge_adaptivity/bridge_lti/urls.py]: Extra one...

File: bridge_adaptivity/bridge_lti/urls.py [hunk 2]
```diff
@@ -4,6 +4,6 @@
...
+    url(r'^launch(?P<collection_id>\d*)$', lti_launch, name='launch'),
```
Comment (idegtiarov) [inline bridge_adaptivity/bridge_lti/urls.py]: That is not correct, slash is required here, please revert.

Comment (wowkalucky) [reply bridge_adaptivity/bridge_lti/urls.py]: But it doesn't catch the `/lti/launch` case.

Comment (idegtiarov) [reply bridge_adaptivity/bridge_lti/urls.py]: Why we need to care about '/lti/launch' when we have `/lti/launch/`?

Comment (wowkalucky) [reply bridge_adaptivity/bridge_lti/urls.py]: Because of 50% of cases, instructors will mistyped it?

Comment (idegtiarov) [reply bridge_adaptivity/bridge_lti/urls.py]: That is not the case, django appends the slash to the end of the URL by default.

**Pull request merged**